packages/rocketchat-ui/lib/chatMessages.coffee

```diff
@@ -330,21 +330,24 @@ class @ChatMessages
 		this.hasValue.set input.value isnt ''
 
 	keydown: (rid, event) ->
+		sendOnEnter = Meteor.user()?.settings?.preferences?.sendOnEnter
 		input = event.currentTarget
 		$input = $(input)
 		k = event.which
 		this.resize(input)
+
 		if k is 13
-			if not event.shiftKey and not event.ctrlKey and not event.altKey and not event.metaKey # Enter without shift/ctrl/alt
-				event.preventDefault()
-				event.stopPropagation()
-				if this.editing.id
-					this.update(this.editing.id, rid, input)
-				else
-					this.send(rid, input)
-				return
-			else if not event.shiftKey
-				return input.value +='\n'
+			if sendOnEnter is 1 || (sendOnEnter is 2 and Meteor.Device.isDesktop())
```

Comment:
How can I send the message when the ender to send is disabled? Shouldn't it be like here (github)?
My opinion:
## Enter send the message
`Enter` sends the message
`Enter + CTRL|META|SHIFT|ALT` adds a line break

## Enter does no send the message
`Enter` adds a line break
`Enter + SHIFT` adds a line break
`Enter + ALT|OPTION|CTRL|META` sends the message


What do you think?